tests/components/google_assistant/test_helpers.py

```diff
@@ -242,3 +243,12 @@ async def test_sync_entities_all(agents, result):
         res = await config.async_sync_entities_all()
         assert sorted(mock.mock_calls) == sorted([call(agent) for agent in agents])
         assert res == result
+
+
+def test_supported_features_string(caplog):
+    """Test bad supported features."""
+    entity = helpers.GoogleEntity(
+        None, None, State("test.entity_id", "on", {"supported_features": "invalid"})
+    )
+    assert entity.is_supported() is False
+    assert "Entity test.entity_id contains invalid supported_features value invalid"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)